test/Libraries/Microsoft.Extensions.Compliance.Abstractions.Tests/Redaction/RedactorTest.cs

```diff
@@ -31,38 +31,38 @@ public static void Redaction_Extensions_Return_Zero_On_Null_Input_Value()
     [InlineData(10000)]
     public static void User_Can_Get_String_From_IRedactor_Using_Extension_Method_With_Different_Input_Length(int length)
     {
-        var data = new string('*', length);
+        string data = new string('*', length);
```

Comment:
why this change?

Reply:
In many .NET repos we don't use `var` for integral types and when the type isn't obvious, see https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md points 10 and 11. 
I rolled back this specific one.